### DISCUSSION TITLE
Error: Get config Error: Command failed with UNKNOWN

### DISCUSSION THREAD

**Discussion by SagarDeoghare** [prisma/prisma#3953] [Q&A] [answered]
npx prisma generate
Environment variables loaded from prisma\.env
Prisma Schema loaded from prisma\schema.prisma
Error: Get config Error: Command failed with UNKNOWN: 
.\node_modules\@prisma\cli\query-engine-windows.exe cli get-config --ignoreEnvVarErrors
spawn UNKNOWN

Getting error while running
npx prisma introspect and 
nps prisma generate

"@prisma/client": "2.8.0",
"@prisma/cli": "2.8.0",

**Comment by ryands17** [comment, marked as answer]
@SagarDeoghare Could you try again with the latest version 2.9.0 by removing `node_modules` and updating Prisma?

**Reply by SagarDeoghare** [reply]
Thanks, It Worked!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
